audformat/core/utils.py

```diff
@@ -865,6 +866,12 @@ def join_labels(
     if not isinstance(labels, list):
         labels = list(labels)
 
+    if any([isinstance(x, str) for x in labels]):
+        raise ValueError(
+            'Cannot join labels from a misc table '
+            'with other label types.'
```

Comment:
This might be misleading for a user who just provided a string `'xyz'` instead `['xyz']` and is not dealing with a misc table at all.

Reply:
So maybe you should repeat the string here, and mention that we assume a misc table is meant, because a string was provided?

Reply:
So how would you rephrase?

Reply:
Maybe along those lines:
```python
raise ValueError(
    f"A string value was provided as scheme: '{x}', "
    "which assumes labels defined in a misc table. "
    "'join_labels()' does not support labels from misc tables."
)
```

Reply:
If we also rely on this error message in `join_schemes()` we need to point out that it is possible if all schemes are misc table schemes. So maybe:

```
['misc'] are IDs of misc tables and cannot be joined with 'join_labels()'. Instead use join_schemes(). However, this requires that all schemes that are joined use labels from a misc table.
```

Reply:
The other option we have is raise different errors in `join_schemes()` and `join_labels()`.

Reply:
> ```
> ['misc'] are IDs of misc tables
> ```

This is not true in all cases, as the user can provide a random string.

Maybe it will be easier to use two different error messages in `join_labels()` and `join_schemes()`.

If you still prefer a single one, I would change to:

```
String values were provided for labels(s): ['misc'],
which assumes those labels are IDs of misc tables.
Labels defined in misc tables cannot be joined with 'join_labels()'.
Instead use 'join_schemes()'.
However, this requires that all schemes that are joined use labels from a misc table.
```

Reply:
I decided to raise two different error messages.

`join_labels()`:

```
The following string values were provided: '['misc_id']'. 
This assumes that labels are defined in misc tables
 with according IDs, which is not supported by 'join_labels()'.
```

`join_schemes()`:

```
Cannot join schemes with labels from a misc table 
with schemes that do not receive their labels from a misc table.
```

Reply:
Great, looks good now.